atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -226,9 +227,16 @@ def manifest_list_entries_match(self, image, build_id):
             v2_digest = archive['extra']['docker']['digests'][v2_type]
             koji_archives_data[arch] = v2_digest
 
+        # Additional platforms might contain v2 digests of different builds
+        for k in list(manifest_list_data.keys()):
+            if k not in koji_archives_data:
+                self.log.warning('Removing missing %s platform in Koji from manifest list', k)
```

Comment:
Could we say "manifest list platform %s is not present in koji archive and will not be veriied"; If it is easily available, we can add the archive id there.

Reply:
Koji archives are platform based so I can't provide ID of non-existing ones, but how about adding koji-build-id which should clearly inform which platforms are expected in manifest list?

```
 self.log.warning('Manifest list platform %s is not present in Koji build and will not be verified. koji-build-id=%s', k, build_id)
```

Reply:
Great!